src/common/config.cc

```diff
@@ -204,8 +204,18 @@ int md_config_t::parse_config_files(const char *conf_files,
 				    int flags)
 {
   Mutex::Locker l(lock);
+
   if (internal_safe_to_start_threads)
     return -ENOSYS;
+
+  if (!cluster.size() && !conf_files) {
```

Comment:
Minor: ```cluster.empty()``` looks cleaner in an ```if``` statement since it returns a bool instead of a size_t being interpreted as a bool